sessions/css-basics/social-card/readme.md

```diff
@@ -0,0 +1,12 @@
+# CSS Basics: Challenges
```

Comment:
I like the outsourcing of the exercise to a readme file. I would prefer the file to be named in all uppercase letters `README.md` as it would stand out more and also follow the conventions for GitHub projects.